examples/process_gain.py

```diff
@@ -31,29 +31,16 @@
 file1 = "240123_power_in_analytic.h5"
 file2 = "240123_power_out_analytic.h5"
 nu_name = r"$\nu$"
-# Nodenames for both files match so extract them here for both files
-all_node_names = sorted(
-    psd.find_file(
-        re.escape(file1),
-        exp_type=data_dir,
-        return_list=True,
-    ),
-    key=lambda x: float(x.split("_")[1]),
-)
-# Frequency of signal saved in each nodename in kHz
-frq_kHz = np.array(
-    [float(j.split("_")[1]) for j in all_node_names]
-)
 
 
 def determine_power_from_fit(filename, guessamp, guessph):
-    """Fit time capture to extract the amplitude ($V_{p}$) for each node within
-    the fed HDF5 file.
+    """Fit time-domain capture to extract the amplitude ($V_{p}$) for each node
+    within the HDF5 file.
 
     Parameters
     ==========
     filename: str
-        Name of HDF5 file
+        Name of HDF5 file --- contains multiple nodes, named according to frequency
```

Comment:
more descriptive since a big portion deals with iterating through the nodes

Reply:
the main issue here was the RST formatting